lib/fog/aws/models/compute/volume.rb

```diff
@@ -36,31 +36,50 @@ def ready?
           state == 'available'
         end
 
-        def save
-          raise Fog::Errors::Error.new('Resaving an existing object may create a duplicate') if persisted?
```

Comment:
No, since if there is an identity, its going to call the modify_volume method instead of create_volume.